_deploy/p/gnoswap/pool/sqrt_price_math.gno

```diff
@@ -53,24 +55,23 @@ func sqrtPriceMathGetNextSqrtPriceFromAmount1RoundingDown(
 
 	if add {
 		if amount.Lte(u256.MustFromDecimal(MAX_UINT160)) {
```

Comment:
This conditional statement is repeated other place as well. By extracting this into a function, you can improve readability and reduce duplication.

ref: https://refactoring.guru/ko/decompose-conditional